RafaelGondi/stock_quote_monitor
# installs flutter

#### What is the purpose of this pull request?
To install flutter

#### How should this be manually tested?
After clonning the project use `flutter run` inside the project folder and see if the flutter demo app will be installed

#### Types of changes
